paper.md

```diff
@@ -374,9 +373,9 @@ This case study showcases the benefits of using Popper: having portable workflow
 
 The adjustments that users need to make to reproduce workflows on Kubernetes and Slurm is described below.
 
-1. To run workflows on Kubernetes clusters, users need to pass some configuration options through a YAML file with contents similar to the one shown below.
+1. To run workflows on Kubernetes clusters, users need to pass some configuration options through a YAML file with contents similar to the one shown in @Lst:kubernetes.
 
-```yaml
+```{#lst:kubernetes .yaml caption="Config file for running on Kubernetes."}
```

Comment:
based on the latest changes we introduced and the ones we plan to work on, registry info won't be necessary anymore, so we can remove it. Only namespace and volume_size will be needed. We can make a note that this is only required if the user is not OK with the defaults